src/routes.c

```diff
@@ -4002,6 +4007,7 @@ WOLFSENTRY_API wolfsentry_errcode_t wolfsentry_route_format_json(
         write_string("\"remote\":{");
 
         if (have_r_attr(SA_REMOTE_ADDR)) {
+#ifndef WOLFSENTRY_NO_STDIO_STREAMS
```

Comment:
these gates aren't right -- `wolfsentry_route_format_json()` should have no stdio deps, and is gated only on `!defined(WOLFSENTRY_NO_JSON) || defined(WOLFSENTRY_JSON_DUMP_UTILS)`.